packages/react-tabs/src/components/Tab/useTabStyles.ts

```diff
@@ -3,56 +3,136 @@ import type { TabSlots, TabState } from './Tab.types';
+    alignItems: 'center',
     ...shorthands.borderColor('none'),
     ...shorthands.borderRadius(tokens.borderRadiusMedium),
-    ...shorthands.borderWidth(tokens.strokeWidthThin),
-    columnGap: tabPendingDesignTokens.gap.medium,
+    ...shorthands.borderWidth(0),
     cursor: 'pointer',
     display: 'flex',
     flexDirection: 'row',
     fontFamily: tokens.fontFamilyBase,
-    fontSize: tokens.fontSizeBase300,
     lineHeight: tokens.lineHeightBase300,
-    ...shorthands.padding(tabPendingDesignTokens.tabPadding.medium),
+    outlineStyle: 'none',
     position: 'relative',
     ...shorthands.overflow('hidden'),
+    textTransform: 'none',
   },
-  horizontal: {
-    alignItems: 'center',
+  mediumHorizontal: {
+    columnGap: pendingSpacingTokens.sNudge,
     justifyContent: 'center',
+    ...shorthands.padding(pendingSpacingTokens.m, pendingSpacingTokens.mNudge),
   },
-  vertical: {
-    alignItems: 'center',
+  mediumVertical: {
+    columnGap: pendingSpacingTokens.sNudge,
     justifyContent: 'flex-start',
+    minWidth: '120px',
+    ...shorthands.padding(pendingSpacingTokens.sNudge, pendingSpacingTokens.mNudge),
   },
-  small: {
-    ...shorthands.padding(tabPendingDesignTokens.tabPadding.small),
-    columnGap: tabPendingDesignTokens.gap.small,
+  smallHorizontal: {
+    columnGap: pendingSpacingTokens.xxs,
+    ...shorthands.padding(pendingSpacingTokens.sNudge, pendingSpacingTokens.sNudge),
+  },
+  smallVertical: {
+    columnGap: pendingSpacingTokens.xs,
+    ...shorthands.padding(pendingSpacingTokens.xxs, pendingSpacingTokens.sNudge),
+  },
+  transparent: {
+    backgroundColor: tokens.colorTransparentBackground,
+    ':hover': {
+      backgroundColor: tokens.colorTransparentBackgroundHover,
+    },
+    ':active': {
+      backgroundColor: tokens.colorTransparentBackgroundPressed,
+    },
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    '& .fui-Tab__icon': {
+      color: tokens.colorNeutralForeground2,
+    },
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    ':hover .fui-Tab__icon': {
+      color: tokens.colorNeutralForeground2Hover,
+    },
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    ':active .fui-Tab__icon': {
+      color: tokens.colorNeutralForeground2Pressed,
+    },
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    '& .fui-Tab__content': {
+      color: tokens.colorNeutralForeground2,
+    },
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    ':hover .fui-Tab__content': {
+      color: tokens.colorNeutralForeground2Hover,
+    },
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    ':active .fui-Tab__content': {
+      color: tokens.colorNeutralForeground2Pressed,
+    },
   },
   subtle: {
     ':hover': {
       backgroundColor: tokens.colorNeutralBackground1Hover,
     },
   },
+  disabled: {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
can we disable this rule for whole file or at least use block disable for particular properties blocks?

Reply:
One nice thing I have found with the per-line is that if the line changes and the rule is no longer violated, the rule suppression gets red-squiggle warning to let you know it isn't needed.  I can try to make it a block suppression though